strr-api/tests/mocks/json/platform_registration.json

```diff
@@ -0,0 +1,63 @@
+{
+  "registration": {
+    "registrationType": "PLATFORM",
+    "registrant": {
+      "firstName": "Test",
+      "lastName": "Test",
+      "phoneNumber": "604-999-9999",
+      "extension": "x64",
+      "faxNumber": "604-777-7777",
+      "emailAddress": "test@test.test"
+    },
+    "platformRepresentatives": [
+      {
+        "firstName": "Test",
+        "lastName": "Test",
+        "phoneNumber": "123-456-8970",
+        "extension": "x64",
+        "faxNumber": "604-777-7777",
+        "emailAddress": "test@test.test",
+        "jobTitle": "Sales Manager"
+      },
+      {
+        "firstName": "Test",
+        "lastName": "Test",
+        "phoneNumber": "604-999-9999",
+        "extension": "x64",
+        "faxNumber": "604-777-7777",
+        "emailAddress": "test@test.test",
+        "jobTitle": "Sales Manager"
+      }
+    ],
+    "businessDetails": {
+      "legalName": "Test Business",
+      "placeOfIncorporation": "Vancouver",
+      "businessNumber": "BN12345",
+      "aliases": [
+        "Business A",
+        "Business B"
+      ],
+      "mailingAddress": {
+        "country": "CA",
+        "address": "12766 227st",
+        "addressLineTwo": "",
+        "city": "MAPLE RIDGE",
+        "province": "BC",
+        "postalCode": "V2X 6K6"
+      }
+    },
+    "platformDetails": {
+      "providers": [
+        {
+          "name": "Provider A",
+          "website": "http://abc.com"
+        },
+        {
+          "name": "Provider B",
+          "website": "http://xyz.com"
+        }
+      ],
+      "hasMoreThanThousandListings": true
```

Comment:
Hopefully the business rule won't change to be a number other than 1000 listings.